en/docs/WIP/overriding-developer-portal-theme.md

```diff
@@ -0,0 +1,421 @@
+              },
+              {
+                "src": "/site/public/images/landing/parallax2.jpg",
+                "title": "Nam vel ex <span>feugiat</span> nunc laoreet",
+                "content": "Nam vel ex feugiat nunc laoreet elementum. Duis sed nibh condimentum, posuere risus a, mollis diam. Vivamus ultricies, augue id pulvinar semper, mauris lorem bibendum urna, eget tincidunt quam ex ut diam."
+              }
+            ]
+          }
+        },
+        "tagWise": {
+          "active": false,
+          "style": "fixed-left",
+          "thumbnail": {
+            "width": 150,
+            "image": "/site/public/images/api/api-default.png"
+          },
+          "key": "-group",
+          "showAllApis": true
+        },
+        "tagCloud": {
+          "active": true,
+          "colorOptions": {
+            "luminosity": "light",
+            "hue": "blue"
+          },
+          "leftMenu": {
+            "width": 200,
+            "height": "calc(100vh - 222px)",
+            "background": "#1a1f2f",
+            "color": "#c7e9ff",
+            "titleBackground": "#335c8b",
+            "sliderBackground": "#335c8b",
+            "sliderWidth": 25,
+            "hasIcon": false
+          }
+        },
+        "social": {
+          "showRating": true
+        },
+        "apiDetailPages": {
+          "showCredentials": true,
+          "showComments": true,
+          "showTryout": true,
+          "showDocuments": true,
+          "showSdks": true,
+          "onlyShowSdks": []
+        },
+        "banner": {
+          "active": false,
+          "style": "text",
+          "image": "/site/public/images/landing/01.jpg",
+          "text": "This is a very important announcement",
+          "color": "#ffffff",
+          "background": "#e08a00",
+          "padding": 20,
+          "margin": 0,
+          "fontSize": 18,
+          "textAlign": "center"
+        },
+        "footer": {
+          "active": true,
+          "text": "",
+          "background": "#bdbdbd",
+          "color": "#222222"
+        },
+        "title": {
+          "prefix": "[Devportal]",
+          "sufix": "- WSO2 APIM"
+        },
+        "languageSwitch": {
+          "active": false,
+          "languages": [{
+            "key": "en",
+              "image": "/site/public/images/flags/en.png",
+              "imageWidth": 24,
+              "text": "English"
+            },
+            {
+              "key": "si",
+              "image": "/site/public/images/flags/si.png",
+              "imageWidth": 24,
+              "text": "'Sinhala'"
+            },
+            {
+              "key": "fr",
+              "image": "/site/public/images/flags/fr.png",
+              "imageWidth": 24,
+              "text": "'French'"
+            }
+          ],
+          "showFlag": true,
+          "showText": true,
+          "minWidth": 60
+        }
+      }
+    }
+  }
+}
+```
+
+## Applying thems for tenant login pages
```

Comment:
typo thems/ themes

Reply:
Closed this PR as the dos file structure is changes now. And added a new PR with requested changes at https://github.com/wso2/docs-apim/pull/854